lib/playload/parser/helpers/create_developers.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Parser
```

Comment:
I don't know if you're seeing this but for some reason the syntax highlighting stops working for me here

Reply:
Yeah I'm seeing that too. Could be the sheer amount of content on the page? I don't think it's a cause for concern.